source/localizable/guides/app/design/bootstrap.en.html.markdown.haml

```diff
@@ -9,13 +9,7 @@ title: Add Bootstrap
 
 %p The app doesn't look very nice yet. Let's do something about that. We'll use the Twitter Bootstrap project to give us nicer styling really easily.
 
-Open the `Gemfile` in your text editor and below the line
-
-= preserve do
-  - code("ruby") do
-    source 'https://rubygems.org'
-
-add
+Open the `Gemfile` in your text editor and add the following line to the list of gems:
```

Comment:
I thought that in this case, saying "add it to the list of gems" can actually cause them to understand better what they are doing... instead of just copy-pasting, they will have to acknowledge that the file contains a list of things, and that they are adding an item to this list. And if they get it wrong - the guide is there to help...
I say let's try it this way and see how it goes in the next event? what do you think?